zilliqa/src/api/mod.rs

```diff
@@ -50,7 +50,7 @@ pub fn rpc_module(
 
 pub fn all_enabled() -> Vec<crate::cfg::EnabledApi> {
     [
-        "admin", "erigon", "eth", "net", "ots", "trace", "txpool", "web3", "zilliqa",
+        "admin", "debug", "erigon", "eth", "net", "ots", "trace", "txpool", "web3", "zilliqa",
```

Comment:
Please can we move the `debug_` methods to a `debug.rs` for consistency and clarity?

Reply:
Thanks :)